data/rohit190183107010.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Rohiit Yaddav",
+  "type": "personal",
+  "bio": "Highly Skilled TechOps Engineer at Datahash",
+  "avatar": "https://github.com/rohit190183107010.png",
+  "links": [
+    {
+      "name": "Follow me on Linkdin",
+      "url": "https://www.linkedin.com/in/rohityj",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/rohit190183107010",
+      "icon": "FaGithubAlt"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/RohitYJ",
+      "icon": "FaTwitter"
+    },
```

Please remove the  **'Comma'** after the 'closing brace **}**' :+1: 
```suggestion
    }
```